toolkit/scripts/pkggen.mk

```
@@ -231,6 +232,7 @@ srpms_archive  	= $(OUT_DIR)/srpms.tar.gz
 
 .PHONY: build-packages clean-build-packages hydrate-rpms compress-rpms clean-compress-rpms compress-srpms clean-compress-srpms
 
+##help:target:build-packages=Build requested .rpm files.
```

```suggestion
##help:target:build-packages=Build requested .rpm files from PACKAGE_(RE)BUILD_LIST=, and IMAGE_CONFIG=.
```

Thoughts? Not sure this is needed.

---

There's probably a balance to be struck here between the tersest reminder of what the targets are called and sufficient context to be able to understand how to use them just from this output. It may help more to include motivating examples and instruct the reader to `(see examples below)`.